admin/app/dfp/DfpApi.scala

```diff
@@ -37,6 +37,14 @@ object DfpApi extends Logging {
     })
   }
 
+  def getAllCustomFields: Seq[GuCustomField] = {
+    val stmtBuilder = new StatementBuilder()
+
+    withDfpSession( session => {
+      session.customFields(stmtBuilder).map(toGuCustomField)
+    })
```

Comment:
neatpicking: You can use `{` only
```
withDfpSession { session =>
    session.customFields(stmtBuilder).map(toGuCustomField)
}
```

or even 
`withDfpSession(_.customFields(stmtBuilder).map(toGuCustomField))`